api/python/provisioner/commands/__init__.py

```diff
@@ -764,7 +764,7 @@ def run(self, targets):  # noqa: C901 FIXME
 
             checker = Check()
             try:
-                check_res = checker.run()
+                check_res = checker.run("swupdate_checks")
```

Comment:
Please, consider to use the constant for "swupdate_checks" string

Reply:
comment is addressed